lxd/db/migration.go

```diff
@@ -271,9 +281,14 @@ func importNodeAssociation(entity string, columns []string, row []any, tx *sql.T
 	stmt := fmt.Sprintf(
 		"INSERT INTO %ss_nodes(%s_id, node_id) VALUES(?, 1)", entity, entity)
 	var id int64
+	var ok bool
 	for i, column := range columns {
 		if column == "id" {
-			id = row[i].(int64)
+			id, ok = row[i].(int64)
+			if !ok {
+				return fmt.Errorf("Failed to convert %s ID to int64", entity)
```

Comment:
as entity is a string %q would be nicer to allow seeing where the string value starts and the error message stops